ckanext/ogdchcommands/logic.py

```diff
@@ -138,7 +138,7 @@ def get_path(id):
         filepath = os.path.join(directory, id[6:])
 
         if filepath != os.path.realpath(filepath):
-            raise logic.ValidationError({'upload': ['Invalid storage path']})
+            raise ValidationError({'upload': ['Invalid storage path']})
```

Comment:
See line 3, where we import this error directly:

```
from ckan.logic import NotFound, ValidationError
```

Reply:
thanks